flang/test/Lower/OpenMP/auto-omp.f90

```diff
@@ -0,0 +1,52 @@
+! RUN: %flang_fc1 -O1 -mllvm --enable-affine-opt -emit-llvm -fopenmp -o - %s \
+! RUN: | FileCheck %s
```

Comment:
The lit tests are usually for testing a single step in the compilation flow (e.g. a single pass, just lowering to HLFIR, etc). Testing only one thing at a time makes it clearer what that thing does, and prevents the test from needing to be updated when unrelated code has changed.

However, I can see the use of an integration test here. Please could you put it in `flang/test/Integration/OpenMP/`.

It is probably also helpful to add a direct test that your affine optimization pipeline is being run. For example something like `flang/test/Driver/mlir-pass-pipeline.f90`.